app/views/how-tos/passing-data-page.html

```diff
@@ -10,91 +10,100 @@
+  {% for condition in data.conditions %}
+    <li>{{ condition }}</li>
+  {% endfor %}
+</ul>" | escape }}</code></pre>
 
-      <p>For a radio or checkbox input you need to use the "checked" function:</p>
-
-      <pre class="app-pre"><code class="language-markup">&lt;input name="know-nhs-number" type="radio" value="Yes" &#123;&#123; checked("know-nhs-number", "Yes") &#125;&#125;&gt;</code></pre>
+      <h3>Showing answers in inputs</h3>
 
-      <h3>Clearing data</h3>
-      <p>To clear the data (for example, at the end of a user research session) click the "Clear data" link in the footer.</p>
+      <p>If a user goes back to a page where they entered data, they would expect to see the answer they gave.</p>
 
+      <p>Most inputs use the <code>value</code> option:</p>
 
-      <h3>Using the data in Nunjucks macros</h3>
+      <pre class="app-pre"><code class="language-markup">{% raw %}{{ input({
+  label: {
+    text: "Full name"
+  },
+  id: "full-name",
+  name: "fullName",
+  value: data.fullName
+}) }}
+{% endraw %}</code></pre>
 
-      <p>Example using the "checked" function in a checkbox component macro:</p>
+      <p>For a radios and checkboxes you need to use the "checked" filter:</p>
 
       <pre class="app-pre"><code>&#123;&#123; checkboxes({
-  "idPrefix": "condition",
-  "name": "condition",
-  "fieldset": {
-    "legend": {
-      "text": "Have you ever had any of these conditions?",
-      "classes": "nhsuk-fieldset__legend--l",
-      "isPageHeading": true
+  idPrefix: "conditions",
+  name: "conditions",
+  fieldset: {
+    legend: {
+      text: "Have you ever had any of these conditions?",
+      classes: "nhsuk-fieldset__legend--l",
+      isPageHeading: true
     }
   },
-  "hint": {
-    "text": "Select all that apply"
+  hint: {
+    text: "Select all that apply"
   },
-  "items": [
+  items: [
     {
-      "value": "Asthma",
-      "text": "Asthma",
+      value: "Asthma",
+      text: "Asthma",
       checked: checked("condition", "Asthma")
     },
     {
-      "value": "Cancer",
-      "text": "Cancer",
-      "checked": checked("condition", "Cancer")
+      value: "Cancer",
+      text: "Cancer",
+      checked: checked("condition", "Cancer")
     },
     {
-      "value": "Lung disease (like emphysema or bronchitis)",
-      "text": "Lung disease (like emphysema or bronchitis)",
-      "checked": checked("condition", "Lung disease (like emphysema or bronchitis)")
+      value: "Lung disease",
+      text: "Lung disease",
+      checked: checked("condition", "Lung disease")
     },
     {
-      "value": "Diabetes",
-      "text": "Diabetes",
-      "checked": checked("condition", "Diabetes")
+      value: "Diabetes",
+      text: "Diabetes",
+      checked: checked("condition", "Diabetes")
     }
   ]
 }) &#125;&#125;</code></pre>
 
+      <h3>Clearing data</h3>
+
+      <p>To clear the data, for example at the end of a user research session), ask the user to close their browser.</p>
```

Comment:
I think this is the easiest current way to recommend clearing data, until https://github.com/nhsuk/nhsuk-prototype-kit/issues/389 has been done?